.github/workflows/ci.yml

```
@@ -2,68 +2,54 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 env:
-  GOPROXY: "https://proxy.golang.org"
-  GO111MODULE: "on"
   TAGS: "-tags=czmq4"
 
 jobs:
-
-  build:
-    name: Build
+  main:
     strategy:
       matrix:
-        go-version: [1.18.x, 1.17.x]
         platform: [ubuntu-latest]
-        #platform: [ubuntu-latest, macos-latest, windows-latest]
+        go-version: [1.19.x, 1.18.x, 1.17.x]
```

it isn't written explicitly in this repo (sorry) but the unwritten policy is to support (and thus test) the same set of Go versions that the Go project supports.
_ie:_ `latest version=N` and `N-1`.

```suggestion
        go-version: [1.19.x, 1.18.x]
```

---

Sure